magicctapipe/scripts/lst1_magic/config.yaml

```diff
@@ -279,3 +277,11 @@ dl2_to_dl3:
     source_name: "Crab"
     source_ra: null  # used when the source name cannot be resolved
     source_dec: null  # used when the source name cannot be resolved
+    
+cloud_correction:
```

Comment:
the changes to this config file are fine, but the file in the meantime was moved to a different location (resources directory), and I'm not sure if this will get merged properly

Reply:
I've now done:
git fetch origin
git merge origin/master
to merge the newest version of master into correction branch.